### DISCUSSION TITLE
Change a created event's calendar from default to another

### DISCUSSION THREAD

**Discussion by brandondeweese** [solspace/craft-calendar#93] [Questions & Answers] [answered]
How do you change an event's calendar from default to something else? Seems like this should be super simple.... but it's not. Unless I'm totally missing something here.

**Comment by kjmartens** [comment, marked as answer]
@brandondeweese Unfortunately this is not easily possible because each calendar can have a different set of fields and layout, thus complicating the process, the same way you can't change regular Craft entries to be in a different section.

That said, it is technically possible to manually adjust your database if you have a bunch to change, especially if your calendars' fields all match. You can do this by changing the `calendarId` column value in the `calendar_events` and `calendar_calendar_sites` database tables. Be sure to take a backup of your site before attempting this though. 🙂
